domain/src/main/java/camp/nextstep/edu/github/domain/Repo.kt

```diff
@@ -0,0 +1,7 @@
+package camp.nextstep.edu.github.domain
+
+data class Repo(
+    val id: Int,
+    val fullName: String,
+    val description: String? = "",
```

Comment:
```suggestion
    val description: String = "",
```
기본값이 존재한다면 non-null로 작성해보면 어떨까요?